python/semantic_kernel/functions/kernel_function.py

```diff
@@ -94,6 +94,44 @@ def from_method(
             stream_method=stream_method,
         )
 
+    @staticmethod
```

Comment:
this needs to be a classmethod and just wrap the logic inside KernelFunctionFromPrompt, instead of here.

Reply:
I would love to see the yaml loading code and prompt template config creation code stay in the kernel method, and then simply call the create_function_from_prompt that already exists in the kernel from the new method you added.

Reply:
@jeffrey-m-johnson, does this make sense? Let us know if you have questions, please.

Reply:
@moonbox3 I think so, yeah.

To sanity check, I'll take the logic here and put it in the `create_function_from_prompt_yaml` function that I have in kernel.py. Then create another function in kernel.py for loading yaml from a path and just call `self.create_function_from_prompt_yaml(path)` to reuse the from-string implementation I already have.